data/json/items/comestibles.json

```diff
@@ -11975,6 +12055,34 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "cooked_pumpkin",
+        "name" : "mashed pumpkin",
+        "weight" : 595,
+        "color" : "light_red",
+        "addiction_type" : "none",
+        "spoils_in" : 36,
+        "stim" : 0,
+        "container" : "can_food",
+        "to_hit" : 0,
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : 3,
+        "healthy" : 2,
+        "addiction_potential" : 0,
+        "nutrition" : 24,
+        "description" : "This is a simple but tasty dish made by cooking the pumpkin pulp and then mashing.",
+        "price" : 50,
+        "material" : "veggy",
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "solid",
+        "charges" : 1,
+        "bashing" : 0,
+        "fun" : 5
```

Comment:
Funny thing I don't think I've ever tried just straight pumpkin, as pumpkin pies are all cinnamon and other things that cause the taste. So not sure what fun value just cooked pumpkin would logically have.